hackathon/mdakits/projects.md

```diff
@@ -0,0 +1,275 @@
+# Project Ideas for MDAKits Hackathon
+
+While at the hackathon you can work on anything you want --- all the
+developers are here to support you to build something amazing. If you
+want some inspiration, here are some ideas that the wider MDAnalysis
+community would really want to see. These ideas are in _random order_.
+
+
+## Trajectory clustering ##
+
+It is common to use unsupervised learning algorithms to cluster a
+trajectory by similarity to discover different states.
+
+A typical approach is to first calculate the RMSD of a structure with
+itself across all trajectory frames, forming a $T \times T$ symmetric
+RMSD matrix where $T$ is the number of frames in the
+trajectory. However, any other [metric][] (or distance) instead of
+RMSD will also work if you can produce a suitable _similarity matrix_.
+
+One can then use any number of [clustering][] algorithms to partition
+the similarity matrix and thus assign different cluster numbers to the
+different frames of the trajectory. 
+
+MDAnalysis implements some clustering methods within the ENCORE module 
+for Ensemble Similarity Calculations  (namely, Affinity Propogation and 
+DBSCAN/KMeans, via scikit-learn; as described 
+[in the docs here](https://docs.mdanalysis.org/stable/documentation_pages/analysis/encore/clustering.html#clustering-frontend-mdanalysis-analysis-encore-clustering-clusteringmethod).
```

Comment:
point to mdaencore here instead?

Reply:
Done in [b52168a](https://github.com/MDAnalysis/UGM2023/pull/14/commits/b52168a1ebf6de8087a2b38fb8069d83726bca21)